src/DistributedFFT.chpl

```diff
@@ -214,9 +214,9 @@ prototype module DistributedFFT {
       const (yDst, xDst, _) = DstDom.localSubdomain().dims();
 
       // Set up FFTW plans
-      var xPlan = setup1DPlan(T, ftType, xDst.size, zSrc.size, signOrKind, FFTW_MEASURE);
-      var yPlan = setup1DPlan(T, ftType, ySrc.size, zSrc.size, signOrKind, FFTW_MEASURE);
-      var zPlan = setup1DPlan(T, ftType, zSrc.size, 1, signOrKind, FFTW_MEASURE);
+      var xPlan = setupPlan(T, ftType, {xDst, zSrc}, parDim=2, 1, signOrKind, FFTW_MEASURE);
+      var yPlan = setupPlan(T, ftType, {ySrc, zSrc}, parDim=2, 1, signOrKind, FFTW_MEASURE);
+      var zPlan = setupPlan(T, ftType, {0..0, zSrc}, parDim=1, 1, signOrKind, FFTW_MEASURE);
```

Comment:
I wonder if it would be cleaner to adjust `setupPlan` to look at the rank of the domain passed in, instead of adding `0..0`. To me, this looks a little odd to be passing in `parDim` when the non-batched plan isn't responsible for the parallelization.

Reply:
Part of the issue here is that `parDim` is serving double duty --- it is the direction that we batch on, but it also specifies the stride for the data. In the case for `z`, we're using just the stride aspect.

I'm going to put this one off for now.... if we do a further cleanup of the initializers, we should revisit these there.